### ISSUE TITLE
ModelClient.batch fails on a big number of items

### ISSUE THREAD

**Issue by aliaksandr-d** [legion-platform/legion#269]
**ModelClient.batch()** fails on a big array (6000+ items) with a next error:
`b'<html>\r\n<head><title>413 Request Entity Too Large</title></head>\r\n<body bgcolor="white">\r\n<center><h1>413 Request Entity Too Large</h1></center>\r\n<hr><center>nginx/1.13.12</center>\r\n</body>\r\n</html>\r\n'`
Request size was 20MB.

**Comment by aliaksandr-d** [comment]
1. I've limit request size in pyserve by 100mb, but it didn't helped. 
2. I've checked that edge has no limits in request size.
3. I've set client_max_body_size in edi to 0 (disabled), problem still exists.

**Comment by kirillmakhonin** [comment]
@aliaksandr-d EDI does not participate in model invocation

**Comment by aliaksandr-d** [comment]
Got an error message after removing request size limit:
```
Exception: Wrong status code returned: 504. Data: <html>
<head><title>504 Gateway Time-out</title></head>
<body bgcolor="white">
<center><h1>504 Gateway Time-out</h1></center>
<hr><center>nginx/1.13.12</center>
</body>
```

**Comment by kirillmakhonin** [comment]
@aliaksandr-d we have decided that proxy timeout should be set to 1min

**Comment by kirillmakhonin** [comment]
Closed by #271
